workers/unity/Packages/com.improbable.gdk.core/GameObjectRepresentation/ReadersWriters/ReaderWriterBase.cs

```diff
@@ -77,6 +77,30 @@ public void Send(TComponentUpdate update)
             }
         }
 
+        /// <summary>
+        ///     Yield authority during soft handover
+        /// </summary>
+        /// <exception cref="InvalidOperationException">
+        ///     Thrown if the authority is not <see cref="Improbable.Worker.Core.Authority.AuthorityLossImminent"/>
+        /// </exception>
+        public void SendAuthorityLossImminentAcknowledgement()
+        {
+            if (!VerifyNotDisposed())
+            {
+                return;
+            }
+
+            if (!EntityManager.HasComponent<AuthorityLossImminent<TSpatialComponentData>>(Entity))
+            {
+                throw new InvalidOperationException(
+                    $"Can not acknowledge authority loss when the Authority state is not " +
```

Comment:
do we need the $ here as well?

Reply:
Strictly probably not. This might actually be better a formatted string rather than an interpolated one.

Reply:
Actually I want to remove the two negatives from this.